.travis.yml

```diff
@@ -12,96 +16,190 @@ addons:
     apt:
         packages:
             - graphviz
-            - texlive-latex-extra
-            - dvipng
```

Comment:
Are we sure that these packages are no longer needed for docs builds?

Reply:
That is what Matteo has done while updating HENDRICS

Reply:
So I suppose it works

Reply:
Yep!